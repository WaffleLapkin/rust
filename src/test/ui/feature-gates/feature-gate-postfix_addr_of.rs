fn main() {
    _ = ().&; //~ error: .& syntax is experimental
}
