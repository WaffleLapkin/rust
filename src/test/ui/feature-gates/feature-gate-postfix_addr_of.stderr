error[E0658]: .& syntax is experimental
  --> $DIR/feature-gate-postfix_addr_of.rs:2:11
   |
LL |     _ = ().&;
   |           ^^
   |
   = help: add `#![feature(postfix_addr_of)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
